tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Christoffel.cpp

```diff
@@ -39,4 +54,136 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.Christoffel.",
   test_christoffel<1, IndexType::Spacetime>(0.);
   test_christoffel<2, IndexType::Spacetime>(0.);
   test_christoffel<3, IndexType::Spacetime>(0.);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(gr::Tags::SpacetimeChristoffelFirstKindCompute<3, Frame::Inertial,
+                                                       DataVector>::name() ==
+        "SpacetimeChristoffelFirstKind");
+  CHECK(
+      gr::Tags::TraceSpacetimeChristoffelFirstKindCompute<3, Frame::Inertial,
+                                                          DataVector>::name() ==
+      "TraceSpacetimeChristoffelFirstKind");
+  CHECK(gr::Tags::SpacetimeChristoffelSecondKindCompute<3, Frame::Inertial,
+                                                        DataVector>::name() ==
+        "SpacetimeChristoffelSecondKind");
+
+  // Check that the compute items return correct values
+  const DataVector test_vector{5., 4.};
+  auto spacetime_metric =
+      make_with_value<tnsr::aa<DataVector, 3, Frame::Inertial>>(test_vector,
+                                                                0.);
+  get<0, 0>(spacetime_metric) = -1.5;
```

Comment:
This might be a good place to use `make_with_random_values` or `fill_with_random_values`, similar to my suggestion in #1478 .

Reply:
changed to using `make_with_random_values` as in #1478